projects/plugins/jetpack/modules/carousel/jetpack-carousel.css

```diff
@@ -1058,6 +1058,27 @@ textarea#jp-carousel-comment-form-comment-field:focus::-webkit-input-placeholder
 	display: inline-block;
 }
 
+.jp-carousel-icon-comments .jp-carousel-has-comments-indicator {
+    display: none;
+    font-size: 12px;
```

Comment:
I noticed while testing this on a theme that sets a really light font weight (`300`) that its style bleeds into this element. Shall we add a `font-weight: normal` to this one, too?

![image](https://user-images.githubusercontent.com/14988353/123726083-abae7100-d8d2-11eb-905a-7b86a9879ff9.png)

(This screenshot is with Maywood theme active)

Reply:
Thanks for testing @andrewserong ! Good tip, will do.